.github/workflows/tests.yml

```diff
@@ -1,11 +1,10 @@
-name: Go
+name: Tests and Codecov
 
 on:
   push:
   pull_request:
-    # edtited - "title, body, or the base branch of the PR is modified"
     # synchronize - "commit(s) pushed to the pull request"
-    types: [opened, synchronize, edited, reopened]
+    types: [opened, synchronize]
```

Comment:
We need edited, for when the base branch of the PR changes.

Reply:
Done.